### DISCUSSION TITLE
Stable-diffusion-cpp-GUI, a VERY simple Tkinter GUI for stable-diffusion.cpp

### DISCUSSION THREAD

**Discussion by Topping1** [leejet/stable-diffusion.cpp#249] [Show and tell]
Hi! just wanted to share a little project I'm working on. It is a simple GUI for stable-diffusion.cpp that (hopefully) only uses standard python libraries. The generated image appears at the right side of the program window (not shown on the screenshot below). You are welcome to try it and leave feedback!

Link:
[Stable-diffusion-cpp-GUI](https://github.com/Topping1/Stable-diffusion-cpp-GUI)

**Screenshot**
![screen](https://github.com/leejet/stable-diffusion.cpp/assets/78745143/5160fbb2-9e81-473e-a6ae-26bfb5a6f6d9)

**Comment by MrGeRoI** [comment]
I would use PyTorch to use SD in Python rather than C++. Don't you want to make C++ application with [stable-diffusion.cpp](https://github.com/leejet/stable-diffusion.cpp)?
